ami_spec.gemspec

```diff
@@ -12,9 +12,10 @@ Gem::Specification.new do |gem|
   gem.summary       = gem.description
   gem.homepage      = 'https://github.com/envato/ami-spec'
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = `git ls-files -z`.split("\x0").select do |f|
+    f.match(%r{^(?:README|LICENSE|CHANGELOG|lib/|bin/)})
```

Comment:
Only put the readme, license, changelog files and the lib and bin directories in the released gem package.